src/OpalCompiler-Core/ArgumentVariable.class.st

```diff
@@ -13,6 +13,11 @@ ArgumentVariable class >> semanticNodeClass [
 	^RBArgumentNode 
 ]
 
+{ #category : #queries }
+ArgumentVariable >> declaringNode [
```

Comment:
I think we are using "defining" word for this kind of meaning. There are definingClass in methods and variables. Would it better to follow it here too?

Reply:
ah, right... we have isDefinition, too, on the AST level (which should use this code). Added to the backlog to rename it.